### PR TITLE
[pt-br] Sync Brazilian Portuguese internationalization strings

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes

* Synchronize the internationalization string list to add the new strings from the English version.
* Reorder strings in the `pt-br.toml` file to alphabetical order.
* Add anchor to the content guide in Portuguese so that the notices link to the right place.

### CODE REVIEW

File: data/i18n/pt-br/pt-br.toml [hunk 1]
```diff
@@ -194,38 +334,20 @@ other = "Objetivos"
 [options_heading]
 other = "Opções"
 
+[outdated_blog__message]
```
Comment (MrErlison) [inline data/i18n/pt-br/pt-br.toml:337]: There are two `_` is that ok?

**Comment by MrErlison** [comment]
Hi @stormqueen1990 

Thank you very much for your contribution. 

Just a small fix.

File: data/i18n/pt-br/pt-br.toml (hunk 1 above)
Comment (stormqueen1990) [reply data/i18n/pt-br/pt-br.toml:337]: Hi there, @MrErlison! 👋 

Yes, the English key also looks like that: https://github.com/kubernetes/website/blob/13a9d24f778372fece6b686300c6bbcbcdd8a6ed/data/i18n/en/en.toml#L337-L338

I can also see that it renders correctly in Portuguese in my preview
(https://deploy-preview-41546--kubernetes-io-main-staging.netlify.app/pt-br/blog/2020/09/02/scaling-kubernetes-networking-with-endpointslices/):
image

**Comment by edsoncelio** [comment]
Thank you!!
/approve

**Pull request merged**